test/pipelinerun_test.go

```diff
@@ -179,44 +175,15 @@ func TestPipelineRun(t *testing.T) {
 	t.Parallel()
 	type tests struct {
 		name                   string
-		testSetup              func(ctx context.Context, t *testing.T, c *clients, namespace string, index int) (map[string]*v1alpha1.PipelineResource, *v1beta1.Pipeline)
+		testSetup              func(ctx context.Context, t *testing.T, c *clients, namespace string, index int) *v1beta1.Pipeline
 		expectedTaskRuns       []string
 		expectedNumberOfEvents int
-		pipelineRunFunc        func(*testing.T, int, string, string, map[string]*v1alpha1.PipelineResource) *v1beta1.PipelineRun
+		pipelineRunFunc        func(*testing.T, int, string, string) *v1beta1.PipelineRun
 	}
 
 	tds := []tests{{
-		name: "fan-in and fan-out",
```

Comment:
question: do we have another test that is e2e testing fan-in and fan-out?

Reply:
I don't think we do. But for this case I tried to restore it but would not have been able to without the resources.

Reply:
Could you elaborate on why this is not possible?
It is possible to fan-in / fan-out in a pipeline, not with PipelineResources anymore but with results -> params it definitely is.

Reply:
Chatted w/ @JeromeJu offline about this, I think we can use a few tasks with params and results to do the fan in/fan out behavior

Reply:
Thanks @dibyom , I replaced it with params and results for fan-in-fan-out test case.